app/Role.php

```diff
@@ -194,4 +194,9 @@ private static function toSelectableCollection(array $items)
 
         return collect($objects);
     }
+
+    public function isSysAdmin()
+    {
+        return in_array($this->name, [self::PRINT_ADMIN, self::INTERNET_ADMIN]);
```

Comment:
Sysadmins are only the internet admins (network admin would be a better name). Print admins are users who can modify the print accounts of user. This usually includes all sysadmins (not necessarily), but can also include the president of the students' council to handle the free pages of the council, and could be also added to some trusted individuals (ie. you roommates, so anyone can pay when you are not in your room, but they are).

Note that most of what I wrote is not in place yet, but this is the general idea.
cc: @kdmnk